pages/getting_started/index.tsx

```diff
@@ -0,0 +1,426 @@
+import Head from "next/head";
+import prisma from "@lib/prisma";
+import { getSession, useSession } from "next-auth/client";
+import { User, UserUpdateInput } from "@prisma/client";
+import { NextPageContext } from "next";
+import React, { useState, useEffect, useRef } from "react";
+import { validJson } from "@lib/jsonUtils";
+import AvailableTimes from "@lib/availability/forms/AvailableTimes";
+import TimezoneSelect from "react-timezone-select";
+import useTheme from "@components/Theme";
+import Text from "@components/ui/Text";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import timezone from "dayjs/plugin/timezone";
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+export const ONBOARDING_INTRODUCED_AT = "1628520977921";
```

Comment:
Here, a value for some date that's 100% before a new user's created_at date. 
And 100% after an existing user's created_at date.